src/smif/model/sector_model.py

```diff
@@ -262,12 +262,18 @@ def extract_obj(self, results):
         pass
 
     def get_region_names(self, region_set_name):
-        """Get the list of region names for ``region_set_name``
+        """Get the unordered list of region names for
+        ``region_set_name``
 
         Returns
         -------
         list
             A list of region names
+
+        Notes
+        -----
+        The order of region names in the list does not match the
+        position in a data array. Use ``data_handle.get_region_names()``
         """
         return self.regions.get_entry(region_set_name).get_entry_names()
```

Comment:
Can we replace this with the new data_handle method? E.g. add data_handle argument, then body of function becomes `return data_handle.get_region_names(region_set_name)`

Reply:
(obviously a breaking change, but probably better than silently failing with data in the wrong order)